### ISSUE TITLE
Can't sort allure results by the test's run

### ISSUE THREAD

**Issue by NeONRAcE** [fescobar/allure-docker-service#198]
Hello, team!

I have a problem with generating allure results for the latest run. Let's imagine the abstract example.

First tests run: I'm launching a test-suite "Auth & Reg" with tests like "Authorization", "Registration", "Password recovery" and other tests (totally - 4 tests)
After the first test launch I see 4 results in allure report
![image](https://user-images.githubusercontent.com/17156684/174678019-cffee939-9609-4c01-8a87-37170ce17007.png)

Second tests run: I'm launching another test-suite with tests like "Personal cabinet: Profile", "Personal cabinet: Courses", "Personal cabinet: Speakers".
After the second launch I see not only three latest results, but I see results from suite "Auth & Reg" (4 tests from the first launch) and my new results with personal cabinet's checks. Finally I get the allure report with 7 results instead of 3 (3 tests from the second launch).
![image](https://user-images.githubusercontent.com/17156684/174679490-196c8171-09fb-4ba3-9c9b-74a07d49041d.png)

To sum up: as a user I expect, that the latest report will contain only launched tests from latest run without any test results from the past test runs

Could you help me with solving this problem?

**Comment by fescobar** [comment]
Clean the results before generating a new report.
